src/libraries/Errors.sol

```diff
@@ -42,6 +42,10 @@ library Errors {
                             SABLIER-MERKLE-FACTORY-BASE
     //////////////////////////////////////////////////////////////////////////*/
 
+    /// @notice Thrown when non-admin address calls {collectFees} without setting the fee recipient to the admin
```

Comment:
"Thrown when an unauthorized address attempts to collect the fees."

Reply:
There is no unauthorised address from the perspective of collect fee. So i don't think it would be accurate to say that.

It only reverts if caller and fee recipient both are not admin.

Reply:
I think if's fine to use "unauthorized" in this circumstance, but I won't belabor this point.

happy to use your version